README.md

```diff
@@ -4,36 +4,13 @@ This provides a discussion forum for use with other MIT applications.
 ## Installation specific to this app
 
 This app uses a similar stack to other mitodl apps (Docker/Django/Webpack). Installation steps that are common to
-all of these apps can be found below, beginning with the [Major Dependencies](#major-dependencies) section. **Those 
+all of these apps can be found below, beginning with the [Major Dependencies](#major-dependencies) section. **Those
 installation steps should be completed before the following steps, which are specific to this app.**
 
  1. Set up a reddit instance for use as a backing store. See the README
  at https://github.com/mitodl/reddit-config for instructions on how
  to set up reddit to work with open_discussions.
- 1. With reddit running in the Vagrant VM, log in and create a `script` application to fill in your `.env` vars.
-    - Navigate to `http://reddit.local/` and click the "Log in or sign up" link on the right side of the screen.
-      - If necessary, create a new account using any credentials (no email needed).
-    - Once logged in, click the "preferences" link on the right side of the screen, then click the "apps" tab.
-    - Click the button to create an app. This will reveal a form.
-    - Select the "script" option for this app. "Name" and "Redirect URL" are required, but the values don't matter. Click
-      "create app" when ready. 
-    - You should see a card with information about the created app. The secret and client id can be copied from here:
-      - Under the app name and "personal use script" subheading is the value for `OPEN_DISCUSSIONS_REDDIT_CLIENT_ID`.
-      - Next to the "secret" label is the value for `OPEN_DISCUSSIONS_REDDIT_SECRET`.
-    - Copy the secret/client id to `OPEN_DISCUSSIONS_REDDIT_CLIENT_ID` and `OPEN_DISCUSSIONS_REDDIT_SECRET` in your 
-      open_discussions `.env` file.
- 1. Base64-encode the client ID and update reddit-config:
-    - Use `printf` to encode the client ID instead of `echo` to avoid encoding the newline (e.g.: `printf "KEY" | base64`)
-    - Add the following lines to `reddit-config/r2/local.update` (create it if it doesn't exist):
-    
-        ```
-        [secrets]
-        generate_refresh_token_client_id = <base64 encoded value here>
-        ```
- 
- 1. cd to `reddit-config` and reload/re-provision the Vagrant VM to update the config with your client ID: 
-    `vagrant reload && vagrant provision`.
- 1. Run the open_discussions container and navigate to the running site in your browser 
+ 1. Run the open_discussions container and navigate to the running site in your browser
```

Comment:
Can you change `open_discussions container` to `the containers for open-discussions`?